src/main/java/org/shanerx/tradeshop/enumys/TradeItemType.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *
+ *                         Copyright (c) 2016-2019
+ *                SparklingComet @ http://shanerx.org
+ *               KillerOfPie @ http://killerofpie.github.io
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *                http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  NOTICE: All modifications made by others to the source code belong
+ *  to the respective contributor. No contributor should be held liable for
+ *  any damages of any kind, whether be material or moral, which were
+ *  caused by their contribution(s) to the project. See the full License for more information.
+ *
+ */
+
+package org.shanerx.tradeshop.enumys;
+
+public enum TradeItemType {
+
+    COST,
+    PRODUCT
+
+}
```

Comment:
it's used in multiple classes outside of that, nesting it would limit access to it if I'm not mistaken. Also I really didn't like having the setting and message section enums nested which is why I did it this way

Reply:
Also put this in IllegalItemList for now since that's the only system that uses it for now.